packages/core/src/awsexplorer/regionNode.ts

```diff
@@ -111,6 +113,11 @@ const serviceCandidates: ServiceNode[] = [
         serviceId: 'cloudcontrol',
         createFn: (regionCode: string) => new ResourcesNode(regionCode),
     },
+    {
+        allRegions: true,
+        serviceId: 'dynamodb',
+        createFn: (regionCode: string) => new DynamoDbInstanceNode(regionCode, new DynamoDbClient(regionCode)),
```

Comment:
nitpick: The pattern for these nodes seems to be initializing the client inside of the constructor with a default, rather than providing it directly

Reply:
updated